### DISCUSSION TITLE
Can opentelemetry_ecto reach version 1.0 (non-RC) given that Otel left RC phase?

### DISCUSSION THREAD

**Discussion by Fs00** [open-telemetry/opentelemetry-erlang-contrib#63] [General]
Hello everyone! 👋
At our company, we are maintaining some OpenTelemetry-based internal libraries. When Otel reached final 1.0 version, we took the opportunity to upgrade many of those libraries to Otel 1.0 to avoid keeping the `-rc` suffix in their versions.
However we also have a couple of libraries depending on `opentelemetry_ecto`, and we'd like to get rid of the `-rc` suffix for those too.
Given that opentelemetry_ecto is not depending on other RC libraries, would it be possible to release a non-RC 1.0 version of opentelemetry_ecto?
Thank you!

**Comment by bryannaegele** [comment]
Yeah, I think we can do so once #49 is merged.

**Comment by bryannaegele** [comment]
https://hex.pm/packages/opentelemetry_ecto/1.0.0

**Reply by Fs00** [reply]
Thank you!
